src/screens/QRScreens/ShowQR.tsx

```diff
@@ -7,6 +7,7 @@ import { StyleSheet } from 'react-native';
 import { SignerType } from 'src/services/wallets/enums';
 import DisplayQR from '../QRScreens/DisplayQR';
 import ShareWithNfc from '../NFCChannel/ShareWithNfc';
+import WalletCopiableData from 'src/components/WalletCopiableData';
 
 function ShowQR() {
```

Comment:
If this is only for PSBT could we name it ShowPsbtQR?

Reply:
Or better ShowPSBTData since it's not only the QR

Reply:
Its used in multiple screens for copying PSBT, xpub, 2fa and signer MFP, so this generic name fine.

Reply:
As far as I can see it's used in 3 places, all for PSBT only, but for it to be generic you'd need to remove the added {<WalletCopiableData title="Transaction (PSBT):" data={data} dataType="psbt" />}

Or just make it PSBT specific which would be easier I think